README.md

```diff
@@ -11,18 +11,39 @@ _Variables in bold are required._
 | AWS_ECR_REPOSITORY          |                    | The AWS ECR repository. **This is only required as GitHub repository secret.** |
 | AWS_REGION                  |                    | The AWS region. **This is also required as GitHub repository secret.**         |
 | AWS_SECRET_ACCESS_KEY       |                    | The AWS access key. **This is also required as GitHub repository secret.**     |
-| CONCURRENCY                 | `16`               | The maximum concurrency when indexing CARs.                                    |
+| CONCURRENCY                 | `8`                | Concurrent batch inserts of blocks.                                            |
+| BLOCKS_BATCH_SIZE           | `10`               | Batch size for blocks ops (insert, publish). 10 is max for SQS, 25 is max for Dynamo |
 | DECODE_BLOCKS               |                    | Set to `true` to decode non raw block information and store then in DynamoDB   |
-| DYNAMO_BLOCKS_TABLE         | `blocks`           | The DynamoDB table where store CIDs informations to.                           |
-| DYNAMO_CARS_TABLE           | `cars`             | The DynamoDB table where store CAR files informations to.                      |
-| DYNAMO_MAX_RETRIES          | 3                  | DynamoDB max attempts in case of query failure.                                |
-| DYNAMO_RETRY_DELAY          | 100                | DynamoDB delay between attempts in case of failure, in milliseconds.           |
-| S3_MAX_RETRIES              | 3                  | S3 max attempts in case of failure.                                            |
-| S3_RETRY_DELAY              | 100                | S3 delay between attempts in case of failure, in milliseconds.                 |
+| DYNAMO_BLOCKS_TABLE         | `v1-blocks`        | The DynamoDB table where store CIDs informations to.                           |
+| DYNAMO_CARS_TABLE           | `v1-cars`          | The DynamoDB table where store CAR files informations to.                      |
+| DYNAMO_LINK_TABLE           | `v1-blocks-cars`   | The DynamoDB table with CARs-blocks links.                                     |
```

Comment:
How about calling this `block-indexes` or something? It is the table that stores the index information (car id, block cid, offset, length). Calling it "link" table and "blocks-cars" is not helpful to my understanding of what it contains.

Reply:
`blocks-cars-link` ?

Reply:
That's the same as now!

Reply:
I don't have a better option. It's a link table between cars and blocks - I'd really avoid name like "index" in a db context

Reply:
It is a link table, but calling it "link" is like naming a variable "string". I agree calling it "index" in DB context is not a good idea. What about `block-offsets`? `block-positions`? `block-locations`? I like that last one because the table tells you the location of a block within a given CAR file.

Reply:
> It is a link table, but calling it "link" is like naming a variable "string"

strongly disagree, "link" describes the function of the table, not the type, anyway happy with your favorite name

Reply:
sorry, I'm not going to do that. From db perspective, the convention for link tables are those: `blocks-cars`, `blocks-link-cars` or so.
I understand that blocks and blocks info are important, but from the db perspective we should describe db tables, none of `blocks-*` names describe the function of the table

Reply:
`blocks-cars-position` is ok for me

Reply:
If the table were just storing 2 foreign keys then I'd be happy with `blocks-cars` since its function is to link data in two tables. My point is, it is not doing _only_ that, it **contains information about blocks**, _and_ 2 foreign keys.

So, its function is not really to link data between two tables - it is a table in it's own right, containing block location data within a CAR file. As such, I'd call it something that describes its function, which is storing block locations.

Anyway, it's just a suggestion. Please name it as you see fit.